tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -82,6 +82,7 @@ cc: cc-cloud-hypervisor-tarball \
 	cc-shim-v2-tarball \
 	cc-virtiofsd-tarball \
 	cc-tdx-kernel-tarball \
+	cc-sev-kernel-tarball \
```

Comment:
Ah, one important thing, by the moment we add this to the tarball, we should also handle cleaning up whatever was added as part of this: https://github.com/kata-containers/kata-containers/blob/b9c0f7fb09a9f89fe370be2382b8cc911ba4a9b7/tools/packaging/kata-deploy-cc/scripts/kata-deploy.sh#L186-L201

Please, double-check that everything you generate is cleaned up there and, if not, add there what has to be cleaned up.

Reply:
Everything ends up under /opt/confidential-containers/share/kata-containers/, so it looks like it taken care of.